packages/client/tests/functional/relationMode/tests_1-to-n.ts

```diff
@@ -613,7 +608,7 @@ testMatrix.setupTestSuite(
             //
             // See issue https://github.com/prisma/prisma/issues/15683
 
-            testIf(isRelationMode_prismaAndSetNull).failing(
+            testIf(isRelationMode_prismaAndSetNull)(
```

Comment:
Do we still need these duplicated tests? or can we remove them if they error like the test with same name but `!isRelationMode_prismaAndSetNull` condition?

Reply:
That should be done now.